src/sources.cpp

```diff
@@ -119,7 +119,7 @@ bool chdir_workspace(const char *chdirto, std::string &wake_cwd, std::string &sr
   return true;
 }
 
-static std::string slurp(int dirfd, bool &fail) {
+static std::string slurp(int dirfd, const char * const * argv, bool &fail) {
```

Comment:
BTW, I think I will never learn to read a type signature that looks like that without having to consult the internet (or something like https://cdecl.org/).